### DISCUSSION TITLE
Pit Commander in Outland, Wrong spawn point

### DISCUSSION THREAD

**Discussion by Sochyo** [azerothcore/azerothcore-wotlk#18571] [General]
Hi, i Notice the Pit Commander in Outland spawn at the wrong place.// And hordes cannoners on Orgrim Hammer in Northrend are moving with the turret when they attack you if you go as an ally, will it be fix?

**Comment by FrancescoBorzi** [comment]
please open a bug report for this, see https://www.azerothcore.org/wiki/contribute#how-to-open-an-issue
